index.html

```diff
@@ -1,215 +1,245 @@
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+      <div class="modal-inner"></div>
+    </div>
 
-            <div class="card port-two">
-                <div class="profile order-1"><img src="image-source/Snapshoot Portfolio (1).png" alt="Snapshoot"></div>
-                <div class="card-port-description order-2">
-                    <h3>Multi-Post Stories</h3>
-                    <div class="profile-description">
-                        <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>2015</p>
-                    </div>
-                    <div class="paragraph-description">
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <ul class="lang">
-                        <li>html</li>
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+    <section class="landing">
+      <header>
+        <nav class="nav-container toolbar-container">
+          <a id="logo" href="#">My Logo</a>
 
-            <div class="card port-three">
-                <div class="profile"><img src="image-source/Snapshoot Portfolio (2).png" alt="Snapshoot"></div>
-                <div class="card-port-description">
-                    <h3>Tonic</h3>
-                    <div class="profile-description">
-                        <h5>CANOPY</h5>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>Back End Dev</p>
-                        <img src="image-source/Counter.png" alt="Counter">
-                        <p>2015</p>
-                    </div>
-                    <div class="paragraph-description">
-                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                    </div>
-                    <ul class="lang">
-                        <li>html</li>
-                        <li class="ruby">Ruby on rails</li>
-                        <li>css</li>
-                        <li>javascript</li>
-                    </ul>
-                    <button class="btn" type="button">See Project</button>
-                </div>
-            </div>
+          <a class="hamburger">
+            <i class="fa fa-solid fa-bars"></i>
+          </a>
```

Comment:
- [ ] As the previous reviewer suggested, kindly ensure that you stick to the Figma design template. If you have noticed the logo and hamburger are not centered vertically ☺️ .

Your design 😎 
![center](https://user-images.githubusercontent.com/99394374/215708811-1baca04a-4d79-4c8e-83e1-73162e25c27e.png)

Figma design 😎 
![center fig](https://user-images.githubusercontent.com/99394374/215708886-cceb56f7-9eb6-4226-a70d-7e50d6e31565.png)

Reply:
Please kindly explain more on logo and menu not being vertically centered. And what's the importance of this shift to the project requirements?

Reply:
I honestly feel like this shouldn't stop my project from being approved as this has passed my reviews already before now, please what's the importance of this to this particular project requirement.

Reply:
Thanks for the review @AlphaNtihinduka